data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,470 @@ mission "Unfettered returning home"
+	on visit
+		dialog `You have reached <planet>, but you either did not bring the Anomalocaris, or some of the cargo was on another ship.`
+	to complete
+		has "ship model: Anomalocaris"
+	on complete
+		dialog `After landing, a group of Unfettered gather around your ship and begin unloading the medical supplies, though strangely they don't seem to be in any hurry. Shortly after they've finished, you receive a rather brief message from the Unfettered you bought the ship from. "Thank you for helping our brethren, <last>. Please accept these <payment>."`
+		payment 1500000
+		log "Delivered a pressing food shipment to the Unfettered Hai using an Anomalocaris, which seems to have pleased them."
+
+
+
+mission "Hai Legacy 3"
+	landing
+	name "Ancient Claim"
+	description "Beat the Lightning Bugs in combat. You only have one day, and becoming disabled will result in losing the fight."
+	source "Firelode"
+	deadline 2
+	to offer
+		has "Hai Legacy 2: done"
+		has "ship model: Anomalocaris"
+	on offer
+		conversation
+			`As you land, you realize your ship is the center of attention. One Unfettered that looks like an engineer comes to speak to you, warmer than usual compared to your previous experiences with the Unfettered.`
+			`	"Welcome. Although I do not know how you have obtained this ship, I imagine some of us will see it as a sign. It is a relic of our past, a past we are quite proud of. As such, most Unfettered warriors will say that you need to prove your right to use that ship to us. Luckily for you, I don't believe in any of that. What matters to me is that this ship is very durable, and it's filled with valuable ancient technology.`
+			`	"I even see technology from those savage Korath that we fought valiantly to obtain!"`
+				to display
+					has "outfit: Korath Detainer"
+			`	Her tone suddenly changes as she raises her voice for everyone to hear. "I, Mera, challenge you! The winner will earn the right to own this honorable relic and its mighty weapons!"`
+			choice
+				`	"What do I get if I win?"`
+					goto win
+				`	"Can't I just sell it to you?"`
+				`	"I bought this ship. It's mine and I shall do what I please with it. I'm leaving now."`
+					goto headingout
+			`	"No, because in the eyes of my brethren that would mean I, too, did not deserve this ship. They would then challenge me for it."`
+			label win
+			`	"You will win the right to access our worlds. In order to keep our respect, this Anomalocaris may not be destroyed or sold, until you prove yourself another way. This is a priceless artifact. Should you refuse to prove your worth, you will no longer be allowed to land here without paying a fee.`
+			`	"Beware, for there are no official rules, other than the fact that you may only use this one ship. So, will you take to the skies, or are you too afraid?"`
+			choice
+				`	"I accept."`
+				`	"I have nothing to prove to you. If you want it, you'll have to chase me down. No duel."`
```

Comment:
```suggestion
				`	"I have nothing to prove to you. This is my ship now."`
```
I think this is too explicit. She has to chase you down? What about just fighting you anyways in a duel? You can say no but they can still fight you, or blow you up even now. So it's better to just keep this generic (with a hint of sass).